src/ray/raylet/scheduling_queue.cc

```diff
@@ -8,8 +8,8 @@ namespace {
 
 // Helper function to remove tasks in the given set of task_ids from a
 // queue, and append them to the given vector removed_tasks.
-void RemoveTasksFromQueue(ray::raylet::TaskState task_state,
-                          ray::raylet::SchedulingQueue::TaskQueue &queue,
+template <typename TaskQueue>
+void RemoveTasksFromQueue(ray::raylet::TaskState task_state, TaskQueue &queue,
```

Comment:
I proposed to make this a method of the `TaskQueue` class , like this : `TaskQueue::RemoveTasks`

Reply:
Again, I'm not trying to refactor the code here, if desired it should be done in a different PR.